statics/js/components/graph-layout.js

```diff
@@ -132,7 +132,9 @@ var LinkLabelLatency = Vue.extend({
       }
 
       const limit = 10;
+      const context = Date.now() - maxClockSkewMillis;
       const query = `G.Flows()` +
+        `.Context(${context})` +
```

Comment:
It won't fix the issue as using limit can remove the flow that we expect. My suggestion was to use Flow.Start since really few minutes removing limit. Like Flows().Has('Start', GT())